### DISCUSSION TITLE
Rust

### DISCUSSION THREAD

**Discussion by ggerganov** [ggml-org/whisper.cpp#310] [Bindings]
General discussion about the `rust` bindings for `whisper.cpp`

**Current state:** https://github.com/tazz4843/whisper-rs

**Comment by sigaloid** [comment]
I think the link there might be a bit wrong :sweat_smile:

**Reply by ggerganov** [reply]
🤣 fixed

**Comment by Rubenisme** [comment]
They moved to: https://codeberg.org/tazz4843/whisper-rs/ 
It's a bit annoying to have to make an account there too to get them to update their bindings.

In general, how can we get community bindings to be automatically updated to the latest whisper.cpp version?
